contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -531,6 +531,33 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
     return s_hotVars.latestSecondaryRoundId;
   }
 
+  /**
+   * @notice check if a report has already been transmitted
+   * @param report the report to check
+   */
+  function existReport(Report memory report) internal view returns (bool exist, uint32 roundId) {
```

Comment:
rename to `_reportExists`

Reply:
Renamed the function.

We have to return the round Id in case it exists to set it if the sender is the secondary proxy.